Core/clim-core/gadgets.lisp

```diff
@@ -713,7 +713,14 @@ and must never be nil.")
 ;;; 30.4.9 The abstract text-editor Gadget
 
 (defclass text-editor (text-field)
-  ()
+  ((ncolumns :reader text-editor-ncolumns
```

Comment:
I'm not sure this will work. The concrete class uses an editor substrate but doesn't have the slots. So this will create a detached copy of the information.

What is the goal here? Just having the initargs accepted?

Reply:
I known, but I think that if the manual said that this gadget have these slots then the abstract class must have them. However they work. If you change `:ncolumns` or `:nline` in the view the gadget change the dimensions.